src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImplTest.java

```diff
@@ -90,7 +90,7 @@ void testReadNoMessage() throws ServiceException {
 
         messageConsumer.readAndProcess();
 
-        verify(messageProcessorService, times(0)).processMessage(any());
+        verify(messageProcessorService, times(0)).processMessage(eq(MESSAGE_CONSUMER_ID), any());
```

Comment:
I would use any(), any() here as we are verifying it is NOT called at all. So better to use vague matchers.

Reply:
Changed